Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -535,7 +535,16 @@ RenderBox* RenderObject::enclosingScrollableContainer() const
     return document().documentElement() ? document().documentElement()->renderBox() : nullptr;
 }
 
-static inline bool objectIsRelayoutBoundary(const RenderElement* object)
+static CheckedPtr<RenderElement> nearestBaselineContextAncestor(const RenderElement* object)
+{
+    for (CheckedPtr container = object->container(); container; container = container->container()) {
```

Comment:
I'd use containingBlock() as I don't think a non-block container should be considered as baseline context